exercises/dominik_arominski/ex02_oo_basics/src/main.cpp

```diff
@@ -0,0 +1,154 @@
+#include <iostream>
+#include <string>
+#include <cmath>
+#include <vector>
+#include <algorithm>
+
+class Shape {
+
+protected:
+  std::string type;
```

Comment:
For the member variables we use `m_` prefix in Mantid.

Reply:
I will take note.